# Define the interface with the left and right motors

float32 img_u
float32 img_v
float32 theta

